Map line items

tkell opened the pull request:
This PR does two things:

- Add a bunch of mappings from ActBlue to Knack in `settings.py`, based on the test data that @jillh510 added.

- Adds logic to return one Knack dictionary per line item from ActBlue, with all the donor information duplicated.  So input shaped like `{"donor": {etc}, "contribution": {etc}, "lineitems": [{1}, {2}]}` will return a list of two Knack dictionaries.  

I am not sure if we can send multiple entries to Knack at once - if not, we can just loop over them.

cc @kyoki and @jillh510, let me know if I did the right things!

kcraybould commented on sync/views.py:
```diff
@@ -9,34 +9,65 @@
     authorized = auth(request)
     if request.method == 'POST' and authorized:
         actblue_data = json.loads(request.body)
+        # This currently returns a list of dictionaries
```
We do need one dictionary per knack request

kyoki commented on sync_data/settings.py:
```diff
@@ -3,7 +3,19 @@
-    'contribution#createdAt': '638',
-    'lineitems#0#amount': '639',
-    'contribution#contributionForm': '643'
+    'contribution#createdAt': '888',
```
settings.py is for prod values - if we're loading stuff into the test db (which is how the application is currently set up), we'll need to add values to test_settings.py

tkell commented:
@kyoki, I matched `test_settings.py` and `settings.py`, and updated the keys based on a spreadsheet from @jillh510.

@kcraybould, I updated `views.py` to send one POST per line item, which may cause timeouts in production, which may or may not cause problems with ActBlue.
